finetuner/tuner/augmentation.py

```diff
@@ -63,18 +73,13 @@ def _vision_preprocessor(
             raise AttributeError(
                 f'Document `blob` is None, loading it from url: {doc.uri} failed.'
             )
+    if normalize:
+        doc.set_image_blob_normalization(channel_axis=default_channel_axis)
+        blob = doc.blob
     if blob.dtype == np.float64:
         blob = np.float32(blob)
     if default_channel_axis not in [-1, 2]:
         blob = np.moveaxis(blob, default_channel_axis, -1)
-    if blob.shape[-1] == 3:  # apply to RGB images.
-        transform = A.Compose(
-            [
-                A.Normalize(),
-                A.ToFloat(),
```

Comment:
I think the problem is that `ToFloat` here is not necessary. Can you debug a bit more?

Reply:
we stick with docarray? this gives me more confident that we're doing the same thing as the common preprocess chain, such as `doc.convert_uri_to_tensor().set_tensor_normlzation().set_channel_axis()..`